app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -765,5 +840,15 @@ $after-button-padding-left: govuk-spacing(5);
 .gem-c-layout-super-navigation-header__width-container {
   @include govuk-media-query($until: "desktop") {
     margin: 0;
+    margin-top: govuk-spacing(6);
   }
+
+  @include govuk-media-query($from: "desktop") {
+    margin: govuk-spacing(5) auto 0;
+    max-width: 960px;
+  }
+}
+
+.gem-c-layout-super-navigation-header__column-header {
+  font-size: 24px;
```

Comment:
If the class is only adding the font size we could consider setting it with `.govuk-heading-m`

Reply:
i think the reason i did it this way is that govuk-heading classes have responsive margins